### PR TITLE
task (tests) Add tests for SourceContext prop and logger injection.

### PR DESCRIPTION
**Pull request by jafin**

### CODE REVIEW

File: test/AutofacSerilogIntegration.Tests/SourceContext/Scenarios/AcceptsLogViaCtor.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            _log = log ?? throw new ArgumentNullException(nameof(log));
+        }
+
+        public void CreateLog()
+        {
+            _log.Information("Hello!");
+        }
+    }
+
+    internal interface IAcceptsLogViaCtor : ILogScenario
```
Comment (nblumhardt) [inline test/AutofacSerilogIntegration.Tests/SourceContext/Scenarios/AcceptsLogViaCtor.cs]: Nit: the project doesn't use explicit private/internal modifiers.

**Comment by nblumhardt** [comment]
Awesome, thanks!

File: test/AutofacSerilogIntegration.Tests/SourceContext/Scenarios/AcceptsLogViaCtor.cs (hunk 1 above)
Comment (jafin) [reply test/AutofacSerilogIntegration.Tests/SourceContext/Scenarios/AcceptsLogViaCtor.cs]: will fix and pr later.

Comment (nblumhardt) [reply test/AutofacSerilogIntegration.Tests/SourceContext/Scenarios/AcceptsLogViaCtor.cs]: No rush, totally minor, just passing on a heads-up. Thanks for all the help!

**Pull request merged**